test/integration/short/control-connection-tests.js

```diff
@@ -92,16 +92,11 @@ describe('ControlConnection', function () {
       var cc = newInstance({ policies: { loadBalancing: new TestLoadBalancing() } });
       utils.series([
         cc.init.bind(cc),
-        function (next) {
-          //wait for all initial events
-          setTimeout(next, 5000);
-        },
-        function (next) {
-          helper.waitOnHost(function () {
-            //noinspection JSCheckFunctionSignatures
-            helper.ccmHelper.stopNode(2);
-          }, cc, 2, 'down', next);
-        },
+        // Don't stop the node until we know it's up.
+        helper.waitOnHostUp(cc, 2),
```

Comment:
I think the change makes sense:
- Up / down host events are the result of a change (ie: was down now it's up), where now you can instantly check it if its up/down and if its not, wait for it.
- Host events are not the result of a control connection status event (at least not always) after [NODEJS-106](https://datastax-oss.atlassian.net/browse/NODEJS-106)